test/basic-buy.ts

```diff
@@ -276,32 +271,37 @@ describe('BasicBuyCommand', () => {
             await expect(executeTrigger(triggerId, targetRatio, triggerData)).not.to.be.reverted
         })
 
-        it('does not recreate the trigger if `continuous` is set to false', async () => {
+        it('clears the trigger if `continuous` is set to false', async () => {
             const executionRatio = toRatio(2.55)
             const targetRatio = new BigNumber(2.53).shiftedBy(4)
             const { triggerId, triggerData } = await createTriggerForExecution(executionRatio, targetRatio, false)
 
             const tx = executeTrigger(triggerId, targetRatio, triggerData)
             await expect(tx).not.to.be.reverted
             const receipt = await (await tx).wait()
-            const events = getEvents(receipt, system.automationBot.interface.getEvent('TriggerAdded'))
-            expect(events.length).to.eq(0)
+            const finalTriggerRecord = await system.automationBot.activeTriggers(triggerId);
+            const addEvents = getEvents(receipt, system.automationBot.interface.getEvent('TriggerAdded'))
+            expect(addEvents.length).to.eq(0)
```

Comment:
maybe add
```
const executeEvents = getEvents(receipt, system.automationBot.interface.getEvent('TriggerExecuted'))
expect(executeEvents.length).to.eq(1)
```
since during the removal we get both removed and executed events